frontend/src/__tests__/cypress/cypress/e2e/pipelines/PipelineRuns.cy.ts

```diff
@@ -198,6 +199,53 @@ describe('Pipeline runs', () => {
         );
       });
 
+      describe('Navigation', () => {
```

Comment:
Are you testing the empty state create run button?

Reply:
added